mission-control/docs/playbooks/Actions/gitops.mdx

```diff
@@ -120,6 +120,43 @@ Files create or delete existing files on the git repo.
 | `path`    | Path to file                                                              | `string` | `true`   | `true`      |
 | `content` | Content of the file. Use the `$delete` keyword to delete an existing file | `string` | `true`   | `true`      |
 
+## Env Vars
+
+GitOps action receives a set of commonly used [env vars](../../reference/playbooks/env-vars). These env vars are extracted by traversing up the Flux Kustomization & Git Repository that created the config resource the playbook is running against. 
+They are
+
+| Field       | description                            |
+| ----------- | -------------------------------------- |
+| `git`       | [GitEnvVar](#git-env-vars)             |
+| `kustomize` | [KustomizeEnvVar](#kustomize-env-vars) |
+
+### Git Env Var
+
+<Fields rows={[
+  {field: "file", description: `Name of the file that created the config resource the playbook is run against. It is extracted from the \`config.kubernetes.io/origin\` annotation. The annotation
```

Comment:
Move the requirements into a Prerequisites admonition, and maybe call the entire section Origin Annotations with examples of the annotations added by Kustomize, and what the corresponding env var would like like